### DISCUSSION TITLE
Transclude arbitary #+BEGIN_X/#+END_X blocks

### DISCUSSION THREAD

**Discussion by jpt4** [nobiot/org-transclusion#227] [Ideas]
I tend to use #+BEGIN_X/#+END_X blocks as my basic unit of text organization, because they can be arbitrarily nested, do not imply any formatting differences, and are easy to parse mechanically by other scripts/tools. To the best of my tests, and interpretation of the docs, org-transclusion cannot target specific blocks such as these. I would be very interested in using such a feature, were it available, and solicit the community for their comments on the same.

**Comment by nobiot** [comment]
I would have thought `#+NAME arbitrary name` could work to point to a specific block? I think I have tested it with tables and quotes. Have you tried this for your purpose?

**Comment by jpt4** [comment]
Thank you, this does work, though at the cost of another line of metadata in the source text.
